spec/index.html

```diff
@@ -283,9 +283,157 @@ <h2>Introduction</h2>
   <section id="basic-concepts" class="informative">
     <h2>Basic Concepts</h2>
 
-    <p>FIXME.</p>
+    <p>
+      To ease writing and collaborating on JSON-LD documents, it is a common practice
+      to serialize them as YAML.
+      This requires not only to register a media type to enable content negotiation
+      of linked data documents in YAML, but even to define the expected behavior of
+      applications that processes these documents, including fragment identifiers and
+      interoperability consideration.
+
+      This is because YAML is more flexible than JSON:
+
+      * it supports different encodings, including UTF-8, UTF-16 and UTF-32;
+      * it supports more data types than JSON;
+      * the structure of its documents, that is named YAML representation graph,
+        is a tree that can have cycles.
+    </p>
```

Comment:
```suggestion
    <p>
      This is because YAML is more flexible than JSON:
    </p>

      <ul>
        <li>YAML supports different encodings, including UTF-8, UTF-16, and UTF-32.</li>
        <li>YAML supports more data types than JSON.</li>
        <li>the structure of a YAML document &mdash; that is, a named YAML representation graph &mdash;
        is a tree that can have cycles.</li>
```